### DISCUSSION TITLE
UOS-AI support to use ollama local models

### DISCUSSION THREAD

**Discussion by marcogutama** [linuxdeepin/developer-center#7714] [Features Request & Ideas | 特性请求 & 头脑风暴]
It would be great if functionality was added to uos-ai to use local Ollama models. In this way, those people who do not have internet or have the AI pages blocked at work (as is my case) can use AI in Deepin.

**Comment by BLumia** [comment]
cc @meiyixiang

**Comment by c4yg70** [comment]
Also, why not use vLLM as it's more efficiency and more hardware-friendly, or support openvino?

**Comment by abskvrm** [comment]
> It would be great if functionality was added to uos-ai to use local Ollama models. In this way, those people who do not have internet or have the AI pages blocked at work (as is my case) can use AI in Deepin.

You definitely can use Ollama with UOS, use the following address http://localhost:11434/v1 I have been using it for months. 🙂
